### PR TITLE
qc import fix

### LINKED ISSUE #194: GWR-Import: Handle empty buildings / dwellings list
Ensure that the import is disabled if no buildings or dwellings can be imported.

### PR DESCRIPTION
**Pull request by velrest**
- fix(import): show a warning if no import data was loaded

Resolves #194

### CODE REVIEW

File: addon/resources/import.js [hunk 1]
```diff
@@ -11,6 +11,8 @@ const IMPORT_MAP = {
   buildingEntrance: "fetchEntrances",
 };
 
+export const NO_DATA = Symbol("no-data");
```
Comment (StephanH90) [inline addon/resources/import.js:14]:
```suggestion
export const NO_DATA_ERROR = Symbol("no-data");
```

File: addon/components/model-form.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { inject as service } from "@ember/service";
 import Component from "@glimmer/component";
 import { tracked } from "@glimmer/tracking";
 import { dropTask } from "ember-concurrency";
-import ImportResource from "ember-ebau-gwr/resources/import";
+import ImportResource, { NO_DATA } from "ember-ebau-gwr/resources/import";
```
Comment (StephanH90) [inline addon/components/model-form.js:6]:
```suggestion
import ImportResource, { NO_DATA_ERROR } from "ember-ebau-gwr/resources/import";
```

File: addon/components/model-form.js [hunk 3]
```diff
@@ -26,15 +26,15 @@ export default class ModelFormComponent extends Component {
...
+    return Boolean(
+      this.args.importQueryParams.instanceId &&
+        this.args.importQueryParams.showImport
+    );
   }
 
-  get showImport() {
-    return Boolean(this.importedData);
+  get noImportData() {
+    return this.import.error === NO_DATA;
```
Comment (StephanH90) [inline addon/components/model-form.js:37]:
```suggestion
    return this.import.error === NO_DATA_ERROR;
```

File: translations/fr.yaml [hunk 4]
```diff
@@ -229,23 +229,6 @@ ember-gwr:
...
-      title: Projet de construction {id}
-      realEstateHeading: Identification de l'immeuble (bien-fonds)
-      clientHeading: Maître d’ouvrage
-
-      diff:
-        existingData: Données existantes dans le RegBL
-        importedData: Données importées
-        apply: Reprendre
-
     modelForm:
```
Comment (StephanH90) [inline translations/fr.yaml:232]: Missing
```
modelForm:
    noImportData: Il n'y a pas de données à importer.
```

(from deepl)

**Review by StephanH90 — CHANGES_REQUESTED** [review]

File: translations/fr.yaml (hunk 4 above)
Comment (velrest) [reply translations/fr.yaml:232]: We dont add french translations yet. We have to get them from an official translation company from kt bern.

**Review by StephanH90 — APPROVED** [review]

**Pull request merged**